### PR TITLE
feat: gauge plot lines and range values

### PR DESCRIPTION
**Pull request by martinkrulltott**
Adds support for plot lines and custom range values for Gauge charts
image

Highcharts demo: https://jsfiddle.net/6dqpuax7/

### CODE REVIEW

**Comment by janhenrikoverland** [comment]
Maybe base/target should also say what the actual value is? @cooper-joe

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
         "classnames": "^2.2.6",
         "d2-utilizr": "^0.2.16",
         "d3-color": "^1.2.3",
-        "highcharts": "^7.1.2",
+        "highcharts": "^7.2.1",
```
Comment (martinkrulltott) [inline package.json:70]: Upgraded to 7.2.1 (latest minor, to avoid 8.0.0) to fix an issue with the plot lines being drawn all the way to the center.

**Comment by cooper-joe** [comment]
> Maybe base/target should also say what the actual value is? @cooper-joe

Yeah I think that would make the target/base lines even more useful. Formatting could be: `Target: 120` and `Base: 30`. If the user changes the label for that line it would still include the value after the user-entered label, e.g. `UserLabel: 300`

This could be a setting, but I think it's useful enough to just include them by default now.

**Comment by martinkrulltott** [comment]
Updated to reflect the comments from @cooper-joe and @janhenrikoverland

**Review by edoardo — APPROVED** [review]
LGTM

**Pull request merged**